srcs/app/http/HttpRequest.cpp

```diff
@@ -80,21 +80,20 @@ bool	HttpRequest::IsValidMethod_(const std::string &method) const {
 	return false;
 }
 
-bool	HttpRequest::ParseUri_(const std::string &raw_request) {
-	const std::size_t	uri_start = offset_;
+bool	HttpRequest::ParseRequestTarget_(const std::string &raw_request) {
+	const std::size_t	request_target_start = offset_;
```

Comment:
What does this `offset_` attribute stands for?

Reply:
The [offset_](https://en.wikipedia.org/wiki/Offset_%28computer_science%29) variable is an index into the raw_string parameter of the Parse* methods.

What do you think about renaming it to, for example, "current_parsing_position"?